### PR TITLE
UI: make pasteup importable in sass

### PR DESCRIPTION
**Pull request by sndrs**
## What does this change?

makes `pasteup.js` usable in sass in UI

## What is the value of this and can you measure success?

one source of trust, usable by all

### CODE REVIEW

**Comment by gustavpursche** [comment]
@sndrs Code looks good to me, but I'm not sure if we should keep the pasteup legacy. It has no proper naming conventions for color names. Why not re-starting with a simple `_variables.scss` file for now?

**Comment by sndrs** [comment]
@gustavpursche yeah that's an interesting thought. current pasteup is actually it's _second_ incarnation, so it would't be so bad to think of this as v3. but maybe a fresh start is a good idea too...

`variables` suggests too many things i think – this file should contain only things which should also be available to sass i think.

**Comment by sndrs** [comment]
@sammorrisdesign do you have any thoughts on this? it's part of testing out a new way of rendering frontend, using JS instead of scala.

i'm kind of imagining pasteup being just a collection of variables rather than the complex set of functions and mixins we have currently. what do you reckon?

**Comment by sndrs** [comment]
(this current `pasteup.js` is just a demo version btw)

File: ui/__config__/webpack.config.js [hunk 1]
```diff
@@ -34,7 +37,14 @@ const config = {
                             // prepended to all sass files
                             data: `
                                 @import '~sass-mq/_mq';
+                                @import 'pasteup';
```
Comment (sndrs) [inline ui/__config__/webpack.config.js]: automatically import pasteup into every sass file

**Comment by sndrs** [comment]
@gustavpursche I think as will the reset PR, this one is really about getting a single source of style settings into the JS and Sass environments. after chatting with sam on slack yesterday, i think we have a bigger conversation around the future of pasteup to consider (as you suggest here too) but we can face that at a later date...

for now, 'pasteup' feels like a useful shorthand for 'style presets' – do you think that's ok or too loaded?

**Review by gu-stav — APPROVED** [review]
I'm ok with that :)

**Pull request merged**